photon-ml/src/main/scala/com/linkedin/photon/ml/util/Utils.scala

```diff
@@ -127,6 +127,22 @@ protected[ml] object Utils {
   }
 
   /**
+    * Fetch the java map from an Avro map field.
+    *
+    * @param record The Avro generic record
+    * @param key The field key
+    * @return A java map of String -> Object
+    */
+  def getMapAvro(record: GenericRecord,key: String, isNullOK: Boolean = false):
+      java.util.Map[String, JObject] = {
```

Comment:
Minor: Couple scala style issues here. There's a space missing after the first comma, and each parameter should have its own newline with 4-space indentation if you need a linebreak, e.g.:

```scala
def getMapAvro(
    record: GenericRecord,
    key: String,
    isNullOK: Boolean = false): java.util.Map[String, JObject] = {
```